CLA_SIGNERS.yaml

```diff
@@ -286,6 +287,9 @@ companies:
       - name: Chris Hupman
         email: chupman@us.ibm.com
         github: chupman
+      - name: Yihong Wang
+        email: yh.wang@ibm.com
```

Comment:
Is @yhwang's email address really without a country prefix? It's different from all other IBM email addresses.

Reply:
Yes, it is correct. Newer IBM addresses don't have the prefix.

Reply:
Thanks for clarification.